app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,72 @@
+class ReservationsController < ApplicationController
+  include Pundit::Authorization
+
+  before_action :set_reservation, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+
+  # GET /reservations or /reservations.json
```

Comment:
please remove unnecessary comments
code should be self documenting
ie the function name describes itself

Reply:
Removed them from controllers :)